components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -211,6 +238,16 @@ export class GithubApp {
         }
     }
 
+    private async getCommitInfo(user: User, repoURL: string, commitSHA: string) {
```

Comment:
I think we're loosing context here: Before, we'd effectively also pass branch here. This part is crucial, but not deduceable just from the repo and revision (same commit can be head of multiple branches).

Reply:
The `context` has that information. The commit info is providing additional user-facing information, that we can show in the UI.